pkg/proxy/iptables/proxier.go

```diff
@@ -241,6 +246,14 @@ func NewProxier(ipFamily v1.IPFamily,
 		}
 	}
 
+	// Be conservative in what you do, be liberal in what you accept from others.
+	// If it's non-zero, we mark only out of window RST segments as INVALID.
+	// Ref: https://docs.kernel.org/networking/nf_conntrack-sysctl.html
+	tcpLiberal := false
+	if val, err := sysctl.GetSysctl(sysctlNfConntrackTCPbeLiberal); err == nil && val != 0 {
```

Comment:
Can both kube-proxy and administrators set it?

Reply:
That's another pr https://github.com/kubernetes/kubernetes/pull/120354 , I do not want to mix things

Reply:
@aojea shouldn't this be dynamic, part of the reconciliation loop?
`nf_conntrack_tcp_be_liberal` can be changed any time after kube-proxy boots up

Reply:
I think it's plausible (and consistent with other sysctl uses in k8s) to not care about that possibility. The "normal" scenario is that the admin would drop a file in `/etc/sysctl.d/` and cause it to be set at boot time.

Reply:
(Though assuming #120354 merges we may want to add comments pointing out that `setupConntrack` needs to get called before `createProxier`.)

Reply:
> shouldn't this be dynamic, part of the reconciliation loop?

I rather prefer not to handle that in the loop since it can generate unstability if something is messing around with the option, each loop may have different rules that is not great :/ ... this is a system wide option, as Dan points out we may assume normal behavior https://github.com/kubernetes/kubernetes/pull/120412#discussion_r1315950908